CMakeLists.txt

```diff
@@ -356,10 +356,11 @@ if (NANOGUI_EXTRA_ICON_RESOURCES)
   endforeach()
 endif()
 
-# Run simple cmake converter to put font files into the data segment
-
-# Glob up resource files
-file(GLOB resources "${CMAKE_CURRENT_SOURCE_DIR}/resources/*.ttf")
+# Begin building the list of fonts to process with bin2c
+# These fonts are vendored with NanoGUI, but additional fonts may be added.
+foreach(font entypo.ttf Roboto-Bold.ttf RobotoMono-Bold.ttf RobotoMono-Regular.ttf Roboto-Regular.ttf)
```

Comment:
FWIW in the previous attempt at fixing whatever somehow broke, there were two things I missed / decided needed to fix

1. `nanogui-resources` target was not an explicit dependency of `nanogui-python-obj` (occasional failure on missing `#include <nanogui/resources.h>`.
2. I don't know for certain, but I think another issue was using `file(GLOB ...)`.  We know exactly which fonts we are embedding, and since it is a `DEPENDS` we want those to be explicit (different generators do different things with `GLOB`, CMake maintainers definitively encourage no `GLOB` unless it is absolutely necessary).

So this finally all works again **at build time**, [even on Ninja // OSX](https://travis-ci.org/wjakob/nanogui/jobs/393922647).

Maintenance note: this is mostly the same as the original `bin2c` layout, but I've kept the arguments as `NANOGUI_OUTPUT_H` etc rather than `OUTPUT_H`.  It seems more appropriate (even though it is not a configure time thing anymore), because what we do to process these files (adding them to the font map etc) makes it fairly inflexible in terms of other projects re-using this script.